assets/prototype/domain/eventEditor/data/queries/priceTypes/useDefaultPriceType.ts

```diff
@@ -1,15 +1,16 @@
 import isEmpty from 'ramda/src/isEmpty';
 import usePriceTypes from './usePriceTypes';
+import { PriceType } from '../../types';
 
 /**
  * A custom react hook for retrieving the default price type object.
  */
-const useDefaultPriceType = () => {
-	const allPriceTypes = usePriceTypes();
-	const percentSurchargePriceTypes = allPriceTypes.filter(
+const useDefaultPriceType = (): PriceType => {
+	const allPriceTypes: PriceType[] = usePriceTypes();
+	const percentSurchargePriceTypes: PriceType[] = allPriceTypes.filter(
 		({ baseType, isPercent }) => baseType === 'SURCHARGE' && isPercent
```

Comment:
This is dope:
![image](https://user-images.githubusercontent.com/1477453/71372420-0324a000-25bd-11ea-9e03-3a3b7aba3e45.png)